frontend/src/app/features/goals/goals.component.scss

```
@@ -1,308 +1,512 @@
+  background: rgba(109, 74, 255, 0.04);
+  border: 1px solid rgba(109, 74, 255, 0.15);
+  border-radius: 12px;
+  padding: 16px;
+  animation: fadeIn 0.3s ease;
+
+  &.at-risk {
+    background: rgba(251, 191, 36, 0.04);
+    border-color: rgba(251, 191, 36, 0.2);
+  }
+
+  @keyframes fadeIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)